specification/redis/resource-manager/Microsoft.Cache/stable/2023-08-01/redis.json

```diff
@@ -2302,6 +2302,18 @@
             "name": "PublicNetworkAccess",
             "modelAsString": true
           }
+        },
+        "updateChannel": {
+          "type": "string",
+          "description": "Optional: Specifies the update channel for the monthly Redis Bits the Redis Cache will receive. Preview contains our latest features. Default value is 'Stable'.",
```

Comment:
Caches using 'Preview' update channel get latest Redis updates "x" weeks ahead of 'Stable' channel caches. Can we guarantee the value of "x" here? Should we say at least "x" week ahead instead?

Reply:
We could, since we're deferring the update by a month. We can say 4 weeks ahead.

Reply:
I think I'll omit it since it's not a certain timeframe on a monthly basis.

Reply:
Will use at least since this can change in the future.